inst/odin/lancelot.R

```diff
@@ -1665,10 +1665,43 @@ update(general_tot) <- new_general_tot
 initial(hosp_tot) <- 0
 update(hosp_tot) <- new_ICU_tot + new_general_tot
 
+## cumulative deaths in hospital by age
 initial(D_hosp_tot) <- 0
 delta_D_hosp_tot <- sum(delta_D_hosp)
 update(D_hosp_tot) <- D_hosp_tot + delta_D_hosp_tot
 
+initial(D_hosp_tot_0_49) <- 0
```

Comment:
Again probably order this `D_hosp_0_49_tot`

Reply:
Done